components/forms/ReviewForm.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-console */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import PaypalButton from './PaypalButton';
+
+class ReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      params: {
+        env: '',
+        client: {
+          sandbox: '',
+          production: '',
+        },
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  async fetchData() {
+    // Fetch auth information to proceed with PayPal sucessfully
+    // We will need to manually update the url once the function exists in production
+    const url = 'https://finalize-donations-page--texasjusticeinitiative.netlify.com/.netlify/functions/paypal_params';
```

Comment:
Could we use a relative URL here instead of an absolute one so that we don't need to update it later?

Reply:
Thanks for the detailed review @nholden! Updated this to use a relative URL.